data-raw/watershed/regional_approximation/mean_wua_description.R

```diff
@@ -0,0 +1,40 @@
+library(tidyverse)
+library(readxl)
+library(glue)
+
+mean_wua_area <- read.csv('data-raw/watershed/mean_wua_values.csv', skip = 1)
+
+
+mean_wua_details <- mean_wua_area %>%
+  select(watershed, channel_area_sqm, watershed_area_sqkm) %>%
+  left_join(cvpiaHabitat::watershed_lengths) %>%
+  filter(species == 'fr', lifestage == 'rearing') %>%
+  mutate(meters = miles * 1609.34, width = channel_area_sqm/meters) %>%
```

Comment:
add spaces between operators `width = channel_area_sqm / meters`, consider renaming width to include units `width_meters`